public/js/components/Editor.js

```diff
@@ -160,20 +164,29 @@ const Editor = React.createClass({
     }
   },
 
+  renderWelcomeBox() {
+    const modifierTxt = Services.appinfo.OS === "Darwin" ? "Cmd" : "Ctrl";
+    return dom.div(
+      { className: "welcomebox" },
+      `Want to find a file? (${modifierTxt} + P)`
+    );
+  },
+
   render() {
     const breakpoints = this.props.breakpoints.valueSeq();
 
     return (
       dom.div(
-        { className: "editor-wrapper devtools-monospace" },
-        dom.div({ className: "editor-mount" }),
+        { className: "editor-wrapper" },
+        dom.div({ className: "editor-mount devtools-monospace" }),
         breakpoints.map(bp => {
           return Breakpoint({
             key: makeLocationId(bp.location),
             breakpoint: bp,
             editor: this.editor && this.editor.codeMirror
           });
-        })
+        }),
+        !this.props.selectedSource ? this.renderWelcomeBox() : null
```

Comment:
I'll try that, I can't remember but it seemed more natural. I think I like the header always showing because it feels more consistent, but I don't need to change that right now. I'll see if it's just as easy to render it in `App`.